themes/custom/droopler_theme/scss/layout/_modifiers.scss

```diff
@@ -509,6 +519,25 @@ $d-p-paddings: map-merge(
   }
 }
 
+@include media-breakpoint-up(lg) {
+  .landing-page-header {
+    .paragraph-sections {
+      .first-paragraph {
+        @each $size, $value in $d-p-paddings-extended {
+          @each $placement in (top) {
+            &.padding-#{$placement}-#{$size} {
```

Comment:
jak mamy tylko `top` to usuńmy to `@each $placement in (top) {` i zróbmy po prostu `&.padding-top-#{$size} {`